Procfile

```diff
@@ -1,5 +1,6 @@
-controller: go run ./cmd/argocd-application-controller/main.go --loglevel debug --redis localhost:6379 --repo-server localhost:8081
-api-server: go run ./cmd/argocd-server/main.go --loglevel debug --redis localhost:6379 --disable-auth --insecure --dex-server http://localhost:5556 --repo-server localhost:8081 --staticassets ../argo-cd-ui/dist/app
-repo-server: go run ./cmd/argocd-repo-server/main.go --loglevel debug --redis localhost:6379
+controller: go run ./cmd/argocd-application-controller/main.go --loglevel debug --forcelogcolors --redis localhost:6379 --repo-server localhost:8081
```

Comment:
I feel --forcelogcolors is only required because we use `goreman`. Instead of introducing one more flag can you please use environment variable?